fireant/tests/mock_dataframes.py

```diff
@@ -1,378 +0,0 @@
-# coding: utf-8
-from collections import OrderedDict
```

Comment:
So good to see this go. This file was one of the main reasons why testing transformers was so painful. It certainly reduced code duplication in tests, but honestly that's not among my top concerns in testing. At least not as much as clarity and simplicity. The kind of test you can fully understand by just looking at a single test function, without having to go back and forth between files, is much more valuable.

Reply:
There is something similar to use as mock data but i used some real data to make it more meaningful